main.cc

```diff
@@ -2377,6 +2385,12 @@ sharded<locator::shared_token_metadata> token_metadata;
             auto drain_view_builder = defer_verbose_shutdown("draining view builders", [&] {
                 view_builder.invoke_on_all(&db::view::view_builder::drain).get();
             });
+            
+            checkpoint(stop_signal, "starting view building worker");
```

Comment:
On line 1755 you have "starting the view building worker" and here you have "starting view building worker". What's the difference? The messages look confusingly similar.